compat/main.py

```diff
@@ -200,6 +189,11 @@ def from_dict(cls, data: Dict):
 
     def _clone(self, working_dir: Path):
         if working_dir.exists():
+            for root, dirs, files in os.walk(working_dir):  
+                for dir in dirs:
+                    os.chmod(os.path.join(root, dir), stat.S_IRWXU)
+                for file in files:
+                    os.chmod(os.path.join(root, file), stat.S_IRWXU)
```

Comment:
Why do we have to make all files executable? That might be dangerous and should not be necessary.

Maybe just make it readable and writable (and not executable):

```suggestion
                for file in files:
                    os.chmod(os.path.join(root, file), stat.S_IRUSR | stat.S_IWUSR)
```